docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,207 @@
+networks, private (like your office/data center) or public (like the
+internet), should be considered insecure. Firewalls and segmented
+networks are not enough. Even application specific networking is not
+enough. The next step is to not trust the network.
+
+This thought process is a new way of conceptualizing network security
+and is referred to as "Zero Trust." Not that trust is not present, far
+from it. Today's networks are the home to distributed software
+solutions. Whether they are high scale mesh networks or small scale
+client servers: there are distributed software running everywhere.
+Distributed systems are rife with trust and trust issues. Zero Trust
+means: "Zero Trust Without Verification." And it turns out that putting
+that trust in pace isn't simple.
+
+Whether you are an encryption expert or a newcomer, welcome! This series
+is for you! It assumes you know nothing and takes you from soup to nuts
+on why this is a problem and how we can apply encryption with a Zero
+Trust model. The model described here is a feature of Ziti. Ziti is part
+of the OpenZiti account on Github. The series starts with the basics and
+dovetails into Ziti's Enrollment system, which bootstraps trust in a
+Zero Trust environment.
+
+The parts are as follows.
+
+- [Part 1: Encryption Everywhere](./part-01.encryption-everywhere.md)
+- [Part 2: A Primer On Public-Key Cryptography](./part-02.a-primer-on-public-key-cryptography.md)
+- [Part 3: Certificates](./part-03.certificates.md)
+- [Part 4: Certificate Authorities & Chains Of Trust](./part-04.certificate-authorities-and-chains-of-trust.md)
+- [Part 5: Bootstrapping Trust](./part-05.bootstrapping-trust.md)
+
+### Trust & Distributed Systems
+
+Developing applications that are composed of distributed pieces of
+software that require network connectivity amongst themselves is a
+challenging problem to solve. Multiple pieces of software must be
+configured with encryption to identify itself and to trust the other
+parts of the software. This process is "bootstrapping trust." It
+involves taking a system with no trust to one that has secure
+connections amongst them that can verify each connection made. It is not
+a simple task. It requires multiple levels of choices that need to be
+compounded together into a secure system.
+
+There are a few choices when it comes to securing those communications.
+Some have higher barriers to entry than others. For example, sharing a
+secret password used to power an encryption algorithm can provide
+security. In general, it is easy to deploy software, either manually or
+automatically, with that secret. However, it is also fundamentally weak
+as there is only one secret in the system necessary to compromise the
+entire system. It also does not individually identify the software in
+the system, nor does it provide any real way to ensure that both sides
+of the communication are not compromised. On top of that, there is no
```

Comment:
fwiw, "On top of that" might be too colloquial for non-native speakers. I'd replace it / reword this sentence. you are also using 'nor' again back to back. that could be a simple "and" however when I read the sentence it feels "run-on-ish". 

perhaps:
"Worse still, there is no way to revoke trust from any individual system as all the systems are sharing the same secret. If all the prior reasons were not enough to sway you away from using a single shared secret then consider this. With a single shared secret there is no easy way to change that secret system-wide and it's not easily abstracted into more secure environments such as with security modules" (honestly i'd probably split that sentence into two as well probably)